ui_curses.c

```diff
@@ -110,18 +110,19 @@ static char *play_queue_autosave_filename;
 /* shown error message and time stamp
  * error is cleared if it is older than 3s and key was pressed
  */
-static char error_buf[512];
+static char error_buf[4096];
 static time_t error_time = 0;
 /* info messages are displayed in different color */
 static int msg_is_error;
 static int error_count = 0;
 
 static char *server_address = NULL;
 
-static char print_buffer[1024];
+static char title_buffer[4096];
```

Comment:
Im not too fond of replacing multiple local variables with a single file global variable.